x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/directive.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+// @ts-ignore
+import { uiModules } from 'ui/modules';
+const module = uiModules.get('apps/ml', ['react']);
+
+import { IndexPattern, IndexPatterns } from 'ui/index_patterns';
+import { I18nContext } from 'ui/i18n';
+import { IPrivate } from 'ui/private';
+
+import { InjectorService } from '../../../../common/types/angular';
+// @ts-ignore
+import { SearchItemsProvider } from '../../../jobs/new_job/utils/new_job_utils';
+// Simple drop-in type until new_job_utils offers types.
```

Comment:
`SearchItemsProvider` has a definition now, so the this type and the `// @ts-ignore` isn't needed anymore.
line 45 can just be `const createSearchItems = Private(SearchItemsProvider)`